services/repository-managers/src/main/java/org/sagebionetworks/repo/util/jrjc/JRJCHelper.java

```diff
@@ -13,23 +14,35 @@
 	private static final String JIRA_PRINCIPAL_ID_ISSUE_FIELD_NAME = "Synapse Principal ID"; // The ID of the Synapse user reporting the issue.
 	private static final String JIRA_USER_DISPLAY_NAME_ISSUE_FIELD_NAME = "Synapse User Display Name"; // The display name of the Synapse user reporting the issue.
 	private static final String JIRA_SYNAPSE_ENTITY_ID_FIELD_NAME = "Synapse Data Object"; // The ID of the Synapse object which is the subject of the issue.
+	private static final String JIRA_COMPONENT_FIELD_NAME = "Components";
+	private static final String JIRA_COMPONENT_NAME = "name";
+	private static final String JIRA_COMPONENT_FIELD_VALUE_RESTRICTION_REQUEST = "Data Restriction Request";
 	private static final String FLAG_SUMMARY = "Request for ACT to review data";
 	private static final String RESTRICT_SUMMARY = "Request for ACT to add data restriction";
 	
 	public static String createFlagIssue(JiraClient jiraClient, String principalId, String displayName, String dataObjectId) {
-        Map<String,String> params = new HashMap<String,String>();
+        Map<String,Object> params = new HashMap<String,Object>();
         params.put(JIRA_PRINCIPAL_ID_ISSUE_FIELD_NAME, principalId);
         params.put(JIRA_USER_DISPLAY_NAME_ISSUE_FIELD_NAME, displayName);
         params.put(JIRA_SYNAPSE_ENTITY_ID_FIELD_NAME, dataObjectId);
         CreatedIssue createdIssue = createIssue(jiraClient, JIRA_FLAG_ISSUE_TYPE_NAME, FLAG_SUMMARY, params);
 		return createdIssue.getKey();
 	}
 	
+	public static JSONArray componentName(String name) {
```

Comment:
Instantiate a `com.fasterxml.jackson.databind.ObjectMapper` for JSON serialization in `JiraClientImpl.searializeBasicIssue`

Reply:
I started down this road but it turns put that the mapping is non-trivial.  I would have to define half a dozen new POJOs to get ObjectMapper to do the right thing.  So I don't think it's worth the effort.